src/Integration/Telemetry/TelemetryManager.cs

```diff
@@ -28,32 +27,19 @@
 
 namespace SonarLint.VisualStudio.Integration
 {
-    [Export(typeof(ITelemetryManager))]
-    [PartCreationPolicy(CreationPolicy.Shared)] // MEF Singleton
-    public sealed class TelemetryManager : ITelemetryManager
+    public sealed class TelemetryManager : ITelemetryManager, IDisposable
```

Comment:
Not worth to do it now but this class could be split into a couple of smaller ones (with smaller responsabilities).

Reply:
Perhaps some other day :)